src/components/alert/alert.js

```diff
@@ -63,9 +63,13 @@ export default {
     dismiss() {
       this.clearCounter()
       if (typeof this.show === 'number') {
-        this.$emit('dismiss-count-down', 0)
+        if (parseInt(this.show) || 1) > 0 {
```

Comment:
There is a closing bracket missing.

Reply:
This is also assuming that the user is `v-model`ing the `show` prop, which is not always the case.

It would be better to check the value of `this.dismissCountDown` to see if it is greater than 0.

Reply:
@tmorehouse Well i tried... Still not seeing any reference to `this.dismissCountDown` and the only reference to `dismissCountDown` in the entire file is a local variable in the `showChanges()` method, as per my previous comment.